kml/edit-kml-ground-overlay/src/main/java/com/esri/samples/edit_kml_ground_overlay/EditKMLGroundOverlaySample.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2018 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.edit_kml_ground_overlay;
+
+import java.io.File;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.Slider;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.StackPane;
+import javafx.scene.paint.Color;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.geometry.Envelope;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.layers.KmlLayer;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.ogc.kml.KmlDataset;
+import com.esri.arcgisruntime.ogc.kml.KmlGroundOverlay;
+import com.esri.arcgisruntime.ogc.kml.KmlIcon;
+import com.esri.arcgisruntime.symbology.ColorUtil;
+
+public class EditKMLGroundOverlaySample extends Application {
+
+  private SceneView sceneView;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene fxScene = new Scene(stackPane);
+      fxScene.getStylesheets().add(getClass().getResource("/style.css").toExternalForm());
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Edit KML Ground Overlay Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(fxScene);
+      stage.show();
+
+      // create a map and add it to the map view
+      ArcGISScene scene = new ArcGISScene(Basemap.createImagery());
+      sceneView = new SceneView();
+      sceneView.setArcGISScene(scene);
+
+      // create a slider for adjusting the rotation
+      Slider slider = new Slider(0, 1, 1);
+      slider.setMaxWidth(300);
+      slider.setShowTickLabels(true);
+      slider.setShowTickMarks(true);
+      // create a label for the slider
+      Label label = new Label("Overlay Opacity: ");
```

Comment:
The text and the slider are a little offset when aligned together on the same row. With this in mind, I'd prefer to see the layout as it is in the design: label above and then the slider below. I think this would look a little neater (and would follow for example the Terrain Exaggeration layout). I'd therefore suggest using a VBox rather than an HBox (might have to do some tweaking to get the start of the text to align with the start of the slider).

Reply:
Having to fiddle around with the veritcal alignment is exactly the reason I've gone for an HBox.
Changed to VBox, you'll see that the slider is vertically aligned with the label, *if* the slider button itself is brought all the way to the right. If it's not in the rightmost position, it will look slightly offset.